public/js/server.js

```diff
@@ -325,3 +325,11 @@ Server.deleteArchive = function (arcId, callback) {
         })
         .catch((error) => LRR.showErrorToast("Error while deleting archive", error));
 };
+
+/**
+ * Sends a UPDATE request for the metadata of the archive ID
+ * @param {*} arcId Archive ID
+ */
+Server.updateTagsFromArchive = function (arcId, tags) {
+    Server.callAPIBody(`/api/archives/${arcId}/metadata`, "PUT", { tags }, `Updated tags from Archive ${arcId}!`, "Error updating tags from archive", null);
```

Comment:
APIBody needs a `FormData` object, you can't just pass the API a random string and expect it to work.  

```suggestion
    const formData = new FormData();
    formData.append("tags", tags);

    Server.callAPIBody(`/api/archives/${arcId}/metadata`, "PUT", formData, `Updated tags from Archive ${arcId}!`, "Error updating tags from archive", null);
```

Reply:
Fixed on new commit.